pandas/io/formats/style.py

```diff
@@ -70,6 +70,15 @@ class Styler(object):
         a unique identifier to avoid CSS collisons; generated automatically
     caption: str, default None
         caption to attach to the table
+    index: bool, default True
+        determines if the index is rendered in the html output
+
+        .. versionadded:: 0.20.2
+
+    hidden_cols: IndexSlice, default None
```

Comment:
i thought hidden_cols was more intuitive of a name, since these columns won't show up. but i'm fine with changing it if it breaks the standard